pkg/kv/kvserver/store.go

```diff
@@ -2531,12 +2662,6 @@ func (s *Store) RangeFeed(
 // scanning ranges. An ideal solution would be to create incremental events
 // whenever availability changes.
 func (s *Store) updateReplicationGauges(ctx context.Context) error {
-	// Load the system config.
-	cfg := s.Gossip().GetSystemConfig()
-	if cfg == nil {
-		return errors.Errorf("%s: system config not yet available", s)
-	}
-
```

Comment:
what in this change obviates the need to do this? It's nice to have some claim that the replicas have been told about their config. FWIW this here is still racy. We don't synchronously wait for the initial scan, do we?